quiche/src/recovery/gcongestion/bbr2/startup.rs

```diff
@@ -31,6 +31,9 @@
 use std::time::Instant;
 
 use crate::recovery::gcongestion::bbr2::Params;
+use crate::recovery::RecoveryStats;
+use crate::recovery::StartupExit;
+use crate::recovery::StartupExitReason;
```

Comment:
Some of these includes seem unused in this commit, but will become used when looking at the full PR.  I recommend squashing on merge since CI / clippy does not run cleanly on each commit.

I would consider squashing commits now.

Reply:
Yep I want to squash commit.. wish that was the default. I can do that since you are done with the review.

Breaking into separate commits was done by eye (for the ease of review) so I am not surprised I missed that.